### PR TITLE
OKTA-349902: Adding basic tests to validate components are rendering

### PR DESCRIPTION
**Pull request by arsalanbadar-okta**
## Description:
- **What's changed?** 
- Adding selenium tests for verifying basic components rendering like header buttons, links, sign in and sign up flow
- **Is this PR related to a Monolith release?** 

### Resolves:

* [OKTA-349902](https://oktainc.atlassian.net/browse/OKTA-349902)

### CODE REVIEW

File: tests/selenium/framework/page-objects/SignUpPage.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+  isSignInLinkPresent() {
+    return this.getSignInLinkElement().isPresent();
+  }
+
+  navigate(url) {
+    const pageLoadElement = SignUpPage.getPageLoadElement();
+    this.load(url, pageLoadElement);
+    this.waitForPresence(pageLoadElement);
+  }
```
Comment (francoislehoux-okta) [inline tests/selenium/framework/page-objects/SignUpPage.js:56]: Should the `SignUp` and `SignIn` pages have a method that does not take an `URL` param in but redirect to the correct page?

**Review by francoislehoux-okta — APPROVED** [review]

**Comment by francoislehoux-okta** [comment]
This is great, please target `fl-developer-docs-3` if you can :)

File: tests/selenium/spec/signin-spec.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+describe('Sign in page check spec', () => {
+  const signInPage = new SignInPage();
+
+  beforeEach(util.itHelper(async () => {
+    await signInPage.navigate();
+  }));
+
+  it('shows all the buttons', util.itHelper(async () => {
+    expect(await signInPage.isSignInWithEmailButtonPresent(), 'expects login with email button to be present').to.equal(true);
```
Comment (vigneshnarasimankumar-okta) [inline tests/selenium/spec/signin-spec.js:18]: we can also extend the test to check if the text of the button is present in dom ( maybe for next time )

File: tests/selenium/spec/signup-spec.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      expect(await signUpPage.getCurrentURL())
+        .to.contain("accounts.google.com");
+    }));
+
+    it('sign in link click takes user to login page', util.itHelper(async () => {
+      await signUpPage.getSignInLinkElement().click();
+      expect(await signUpPage.getCurrentURL())
+        .to.contain("login");
+    }));
+
```
Comment (vigneshnarasimankumar-okta) [inline tests/selenium/spec/signup-spec.js:55]: We can add another functional test here to see if the text fields especially email fields and country field works as expected and in the end checking if the I agree check box shows up and possibly if clicking sign up takes us to a thank you page.

Comment (arsalanbadar-okta) [reply tests/selenium/spec/signup-spec.js:90]: I have added agree check box and buttons text tests. for actually clicking sign up, I will be creating separate jira as that includes interaction with captcha and might include some investigation

**Review by vigneshnarasimankumar-okta — APPROVED** [review]

**Comment by vigneshnarasimankumar-okta** [comment]
LGTM !

**Pull request merged**